journal-writer/src/main/scala/akka/persistence/journal/writer/WriteJournalAdapterCameo.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.persistence.journal.writer
```

Comment:
Only because akka.persistence.JournalProtocol is private ie. 'internal API'. It uses Jamie Allen's Cameo pattern which is handy for multiple things, as it turns out, also a bridge between packages.